lib/t/feeds.ex

```diff
@@ -41,15 +41,15 @@ defmodule T.Feeds do
   end
 
   defp notify_subscribers(:mode_change, event) do
-    broadcast_from(mode_change_topic(), {__MODULE__, [:mode_change, event]})
+    broadcast(mode_change_topic(), {__MODULE__, [:mode_change, event]})
```

Comment:
As this `notify_subscribers(:mode_change, event)` is called from a non-channel process, I think it's ok to `broadcast` it instead of `broadcast_from` since the latter is mostly useful when we don't want the broadcasted to receive their own broadcast.